lib/resourceread/apiext.go

```diff
@@ -8,12 +8,17 @@ import (
 )
 
 var (
-	apiExtensionsScheme = runtime.NewScheme()
-	apiExtensionsCodecs = serializer.NewCodecFactory(apiExtensionsScheme)
+	apiExtensionsScheme   = runtime.NewScheme()
+	apiExtensionsCodecs   = serializer.NewCodecFactory(apiExtensionsScheme)
+	apiRegistrationScheme = runtime.NewScheme()
+	apiRegistrationCodecs = serializer.NewCodecFactory(apiRegistrationScheme)
 )
 
 func init() {
-	if err := apiextv1beta1.AddToScheme(apiExtensionsScheme); err != nil {
+	//if err := apiextv1beta1.AddToScheme(apiExtensionsScheme); err != nil {
```

Comment:
can we removed commented code

Reply:
will uncomment this, since we need this for crds